### PR TITLE
bt-118: Remove salt

### PR DESCRIPTION
**Pull request by ptarho**
To make changes in the DB you need to run "npm run migrate:dev" after the merge.
Sign-in/sign-up functiononal is still working after changes:
 
Screenshot_1
Screenshot_2
Screenshot_3

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
@ptarho In your PR a lot of different changes not related to SALT

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Can not review PR, because a lot of not related changes

**Comment by ptarho** [comment]
This branch is based on sign-up/sign-in branches. There will be much less changes after they will merge into development. Should I wait for them to be merged?

**Comment by ptarho** [comment]
@nikita-remeslov Or should I merge this changes into sign-up  branch?

File: backend/src/migrations/20230827100608_remove_password_salt.ts
Comment (MaksymGavatuk) [inline backend/src/migrations/20230827100608_remove_password_salt.ts]: Create 
const ColumnName = {
    PASSWORD_SALT: 'password_salt',
};

and use it instead, because you use it in two different places

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

File: backend/src/migrations/20230827100608_remove_password_salt.ts
Comment (ptarho) [reply backend/src/migrations/20230827100608_remove_password_salt.ts]: Done

**Review by nikita-remeslov — APPROVED** [review]

**Review by MaksymGavatuk — APPROVED** [review]

**Pull request merged**
